erdpy/wallet/core.py

```diff
@@ -10,7 +10,7 @@
 BIP39_SALT_MODIFIER = "mnemonic"
 BIP39_PBKDF2_ROUNDS = 2048
 BIP32_SEED_MODIFIER = b'ed25519 seed'
-ELROND_DERIVATION_PATH = [44, 508, 0, 0]
+MULTIVERSX_DERIVATION_PATH = [44, 508, 0, 0]
```

Comment:
Can be `HD_WALLET_DERIVATION_PATH` or `BIP39_WALLET_DERIVATION_PATH`.

Reply:
Changed to `BIP39_WALLET_DERIVATION_PATH`